core/server/master/src/main/java/alluxio/master/file/replication/ReplicationChecker.java

```diff
@@ -219,6 +220,12 @@ private void check(Set<Long> inodes, ReplicationHandler handler, Mode mode) {
         LOG.warn("Unable to complete the replication check: {}, will retry later.",
             e.getMessage());
         return;
+      } catch (ResourceExhaustedException e) {
```

Comment:
Can we combine this with the `JobDoesNotExistException` case?

Reply:
Yeah we can. I think we should also log the exception message though with it so that we don't lose information about which resource is exhausted